CodeFormatter/formatoptions.h

```diff
@@ -112,6 +112,16 @@ enum PHPFixserFormatterStyle {
     kPcfConcatSpaceOne = (1 << 19),
     kPcfEmptyReturnStrip = (1 << 20),
     kPcfEmptyReturnKeep = (1 << 21),
+    kPcfCombineConsecutiveUnsets = (1 << 22),
+    kPcfLinebreakAfterOpeningTag = (1 << 23),
+    kPcfMbStrFunctions = (1 << 24),
+    kPcfNoBlankLinesBeforeNamespace = (1 << 25),
+    kPcfNoMultilineWhitespaceBeforeSemicolons = (1 << 26),
+    kPcfNoNullPropertyInitialization = (1 << 27),
+    kPcfNoPhp4Constructor = (1 << 28),
+    kPcfNoShortEchoTag = (1 << 29),
+    kPcfNoUnreachableDefaultArgumentValue = (1 << 30),
+    kPcfNoUselessElse = (1 << 31),
```

Comment:
Hmm while this works nicely for expanding the enum limit, it then results in another issue, setting the options in the dialog failes to compile as wxVariant isn't able to handel the 64bit int.

I'll work on splitting the config up. Maybe I can make a seporate config object for php-cs-fixer, it would make sense to save in the json config format as that is what php-cs-fixer takes as an input and so i would be able to reuse more code instead of all this format range-ling.

Of cause this would break comparability with the options that are currently in this PR, so if you prefere not to have a break like that you can wait with merging till I have the new format down.